src/memory_region.rs

```diff
@@ -754,41 +545,39 @@ impl<'a> MemoryMapping<'a> {
     ///
     /// Works across memory region boundaries.
```

Comment:
This comment needs an update if we have stopped supported loads across region boundaries.